### ISSUE TITLE
Filtro desativa apos recarga de informação na Grid

### ISSUE THREAD

**Issue by rrgoncalve** [macgile/DataGridFilter#14]
Ao usar o datafilter apos qualquer edição ou recaregar os dados da grid os fintros não funcionam mais .. tenho que reiniciar o programa pra voltar a funcionar. 

não consegui indentificar o porque do fato estar acontecendo.

**Comment by macgile** [comment]
Você faz as modificações diretamente no datagrid ou modifica a coleção por algum outro meio, você tem um exemplo de recarregar os dados no aplicativo demo.

**Comment by rrgoncalve** [comment]
Tenho a função abaixo que recarrega a gride

private void LoadeDados(string search)
        {
            
            DataGridList.ItemsSource = null;            
            DataGridList.ItemsSource = con.GetCategoria(search);           

        }

o pacote adicionei pelo nuget.. e segui os passos como orientado no repositorio.


________________________________________________________________________________________

**Adicione FilterDatagrid ao seu xaml:

Namespace

< Window  xmlns : control = " http://filterdatagrid.control.com/2021 " ...
Ao controle

< control : FilterDataGrid  
 FilterLanguage = " English "  DateFormatString = " d "  ShowStatusBar = " True "  ShowElapsedTime = " False " ...
Propriedades
ShowStatusBar : exibe a barra de status , padrão: falso
ShowElapsedTime : exibe o tempo decorrido de filtragem na barra de status , padrão: falso
DateFormatString : formato de exibição de data , padrão: "d"
FilterLanguage : tradução para o idioma disponível , padrão: inglês
Se você adicionar colunas personalizadas, deve definir AutoGenerateColumns = "False"

TextColumn personalizado

< control : FilterDataGrid .Columns>   
    < control : DataGridTextColumn  IsColumnFiltered = " True " ...
Custom TemplateColumn

A propriedade FieldName de DataGridTemplateColumn é necessária

< control : FilterDataGrid .Columns>   
    < control : DataGridTemplateColumn  IsColumnFiltered = " True "
	                             FieldName = " LastName " ...**

**Comment by macgile** [comment]
Você precisa acionar um evento que informe ao datagrid que a propriedade ItemSource foi alterada, por isso não há atualização do datagrid.

Use INotifyPropertyChanged para isso

**Comment by rrgoncalve** [comment]
https://user-images.githubusercontent.com/19882068/116788706-80013d00-aa81-11eb-8382-716e22e2849b.mp4

Isso que esta acontecendo ... achei que era so no meu projeto mais no projeto -master tambem está assim

**Comment by macgile** [comment]
Sim, é estranho, não há informações no console do Visual Studio no modo de depuração?
Verifique sua versão do framework, ela não deve ser inferior a 4.6.
